examples/STM32L475VG-DISCOVERY-IOT/BTLE_sensors_tof_demo/BTLE_sensors_tof_demo.ino

```diff
@@ -0,0 +1,427 @@
+/*
+
+ BTLE_sensors_tof_demo
+
+ This sketch provides a default demo to be used on STMicroelectronics Discovery L475VG IoT board.
+ It will use several components of the board :
+   * Low energy Bluetooth (SPBTLE_RF)
+   * temperature and pressure sensor (LPS22HB)
+   * temperature and humidity sensor (HTS221)
+   * Time Of Flight sensor (VL53L0X)
+ You can find more information on this board here :
+   http://www.st.com/en/evaluation-tools/b-l475e-iot01a.html
+
+ This sketch will launch 3 services on BLE : Acc, Environnemental and Time.
+ For testing the sketch, you can download on the playstore the "BlueNRG" application provided by
+ STMicroelectronics. (https://play.google.com/store/apps/details?id=com.st.blunrg for Android or
+ https://itunes.apple.com/fr/app/bluenrg/id705873549?mt=8 for Apple)
+ Enable Bluetooth and launch the application on your smartphone. Connect it to the BLueNRG device.
```

Comment:
FIrst need to compile and download the example to the STM32 board right ? 
No other action on the board before you can find the device in the application ?

Reply:
yes, need to compile and download... of course ;)
No other action needed on the board. (Except move in front of the vl53l0x sensor to move the cube.)

Reply:
just would be good to mention it to know what needs to be started first